tests/integration-tests/tests/create/test_create.py

```diff
@@ -207,8 +207,7 @@ def test_cluster_creation_with_invalid_ebs(
     expected_cfn_failure_reason = (
         "Failed to mount EBS volume. Please check /var/log/chef-client.log in the head "
         "node, or check the chef-client.log in CloudWatch logs. Please refer to "
-        "https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html#"
-        "troubleshooting-v3-get-logs for more details on ParallelCluster logs."
```

Comment:
Why this change?

Reply:
I didn't make the change.  I am just fixing the test to match this

Reply:
There isn't any changes regarding this message https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/resources/head_node/user_data.sh#L137

Are you sure this change is necessary?